core/lib/Thelia/Config/I18n/en_US.php

```diff
@@ -47,7 +47,7 @@
     'All Products' => 'All Products',
     'All countries' => 'All countries',
     'All shipping methods' => 'All shipping methods',
-    'An error happend while copying %prev to %dest' => 'An error happend while copying %prev to %dest',
+    'An error happend while copying %prev to %dest' => 'An error happened while copying %prev to %dest',
```

Comment:
It would be great to change the key too, in this file and where this key is used in translate functions.